Library/Formula/erlang.rb

```diff
@@ -39,11 +39,13 @@ class Erlang < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "libtool" => :build
-  depends_on "openssl"
   depends_on "unixodbc" if MacOS.version >= :mavericks
   depends_on "fop" => :optional # enables building PDF docs
   depends_on "wxmac" => :recommended # for GUI apps like observer
 
+  depends_on "libressl" => :optional
+  depends_on "openssl" unless build.with? "libressl"
```

Comment:
We're aware of the fact it will install OpenSSL if you don't have it installed - Yes. But it *won't* use OpenSSL during the compile of the formula itself. We know the ` :recommended ` and ` :optional ` dance on LibreSSL/OpenSSL is imperfect, but LibreSSL isn't yet popular enough to merit rewriting a chunk of Homebrew's core code to handle the use case.

OpenSSL is consistently one of our most downloaded bottles, every month - Quite often *the* most downloaded bottle - You'll find it quite hard to use Homebrew and not have our OpenSSL installed at some point, and I note the same is true for MacPorts. The use case of people who only want to use LibreSSL and refuse to touch OpenSSL at all is tiny at this point.